### ISSUE TITLE
VM doesn't start if aggressive  tree shaker is used

### ISSUE THREAD

**Issue by dkimitsa** [dkimitsa/robovm-libcore10-smoketest#2]
if `<treeShaker>aggressive</treeShaker>` is configured VM fails to boot.  
Crash log:
```
rvmGetMethods + 8337964 (class.c:1104)
findMethod + 8438124 (method.c:60)
findMethod + 8438124 (method.c:60)
getMethod + 8406288 (method.c:73)
rvmGetMethod + 8406752 (method.c:142)
rvmGetInstanceMethod + 8406104 (method.c:167)
rvmThrowNew + 8352060 (exception.c:99)
rvmThrowNoSuchMethodError + 8352940 (exception.c:167)
rvmGetMethod + 8406804 (method.c:145)
RegisterNatives + 8462660 (native.c:176)
_JNIEnv::RegisterNatives(_jclass*, JNINativeMethod const*, int) + 86372 (jni.h:977)
jniRegisterNativeMethods + 86048 (JNIHelp.cpp:78)
jniRegisterNativeMethods(_JNIEnv*, char const*, JNINativeMethod const*, int) + 2654584 (JNIHelp.h:195)
register_java_math_NativeBN + 2665544 (java_math_NativeBN.cpp:569)
luniRegister + 2653956 (Register.cpp:37)
rvmRTInit + 21168 (rt.c:148)
rvmStartup + 8361320 (init.c:358)
bcmain + 8285808 (bc.c:91)
main + 8285680 (bc.c:103)
start + 4
```

**Comment by LordTylus** [comment]
Hey there,

are there any news regarding this issue, and also regarding the missing reflection APIs?

**Comment by dkimitsa** [comment]
@LordTylus  
hi, aggressive tree shaker was fixed a bit in [build #3 ](https://github.com/robovmx/robovmx/releases/tag/x2-libcore12-v10.2.2-20220819), closing. please open any new issue  at [robovmx](https://github.com/robovmx/robovmx/issues) issue page
